add location marker and PictureMarkerSymbol for GeoHazards

nicolaimich opened the pull request.

gruble commented:
Dette tenkte vi sikkert ikke på, men jeg tror vi må løse det: Markøren må nok 
flyttes slik at det er spissen av markøren som "peker på" lokasjonen. Se vedlagte sammenligning mellom gammel og ny løsning.
![marker-posiion](https://user-images.githubusercontent.com/71138449/116994846-8f7ac300-acd9-11eb-9107-1055b367c658.PNG)

nicolaimich commented:
Bra sett. Det kan nok løses med yoffset til PictureMarkerSymbol tipper jeg.

gruble commented on src/app/modules/map/core/classes/regobs-geohazard-marker.ts:
```diff
@@ -1,227 +1,42 @@
  * TODO: Should we use a map service to show observations instead of using this?
  */
-export class RegobsGeoHazardMarker extends L.DivIcon {
+export class RegobsGeoHazardMarker {
```
Her er det mye kode som ikke blir brukt. Trenger dette å være en egen klasse lengre eller kunne man like gjerne flytta de to funksjonene som brukes inn i MapItemMarker?

gruble commented on src/app/modules/registration/components/set-location-in-map/set-location-in-map.component.ts:
```diff
@@ -197,7 +197,11 @@ export class SetLocationInMapComponent implements OnInit, OnDestroy {
     return new Graphic({
       geometry: point,
       //TODO: Få SVG-til å virke! symbol: new SimpleMarkerSymbol({ path: symbolPath })
```
Den TODO'en har du vel fiksa?-)

nicolaimich commented:
- Added 4 SVG icons for outline
- Removed RegobsGeoHazardMarker class. Place two of the functions inside map-item-marker
- Add yoffset and TODO comments
